app/graphql/types/chapter.rb

```diff
@@ -0,0 +1,14 @@
+class Types::Chapter < Types::BaseObject
+  implements Types::MediaUnitInterface
+  description 'A chapter of a Manga'
+
+  field :id, ID, null: false
+
+  field :volume_number, Integer,
+    null: false,
+    description: 'The volume this chapter is related to'
```

Comment:
`volume_number` is unused now that we have a Volume model.  Better to expose `field :volume, Types::Volume` instead, and let API users just access `volume { number }`

Reply:
sure, will add this.

Reply:
Could you remove the `volume_number` field entirely?  I want to remove some of the older stuff in the switch to GraphQL